<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>761</width>
      <height>371</height>
     </rect>
    </property>
    <widget class="QPushButton" name="addAccount">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add account</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deletePassword">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modifyPassword">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Modify password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showAllPassword">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Show all account</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="Result_Display">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>311</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_InputUsername">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>170</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_InputPassword">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>240</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="OKButton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>310</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>150</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>220</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Quit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>310</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Password's Manager</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1019</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
